data/alUSD/data.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "Alchemix USD",
+  "symbol": "alUSD",
+  "decimals": 18,
+  "description": "A synthetic token for users to borrow their future yield.",
+  "website": "https://alchemix.fi",
+  "twitter": "@AlchemixFi",
```

```suggestion
  "twitter": "@AlchemixFi",
  "noBridge": true,
```

I think you might need this if your token doesn't use the standard bridge.  If it does use the standard bridge this is not necessary